public/index.html

```diff
@@ -16,11 +18,145 @@
+                        <input type="text" class="form-control" id="#">
+                    </div>
+                    <div class="form-group col-md-4">
+                        <label for="text">Estimated Time</label>
+                        <input type="number" class="form-control" id="#">
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group col-md-4">
+                        <label for="input">Category</label>
+                        <input type="text" class="form-control" id="#" list="category">
+                        <datalist id="category">
+                            <option>Bussiness</option>
+                            <option>Development</option>
+                            <option>Marketing</option>
+                        </datalist>
+                    </div>
+                    <div class="form-group col-md-4">
+                        <label for="text">Priority</label>
+                        <input type="text" class="form-control" id="#" list="Priority">
+                        <datalist id="Priority">
+                            <option>High</option>
+                            <option>Medium</option>
+                            <option>Low</option>
+                        </datalist>
+                    </div>
+                    <div class="form-group col-md-4">
+                        <label for="text">Difficulty</label>
+                        <input type="text" class="form-control" id="#" list="Difficulty">
+                        <datalist id="Difficulty">
+                            <option>High</option>
+                            <option>Medium</option>
+                            <option>Low</option>
+                        </datalist>
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group col-md-4">
+                        <label for="inputState">Status</label>
+                        <input type="text" class="form-control" id="#" list="statusCheck">
+                        <datalist id="statusCheck">
+                            <option>Ice Box</option>
+                            <option>Ready to start</option>
+                            <option>Under Progress</option>
+                            <option>Blocked</option>
+                            <option>Completed</option>
+                        </datalist>
+                    </div>
+                </div>
+
+                <button type="submit" class="btn btn-primary mt-2 mb-5">Submit</button>
+            </form>
+        </div>
+    </div>
+
+    <!-- JS, Popper.js, and jQuery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
```

Comment:
the jquery is added more then once, please check if and try to use one

Reply:
i will remove which link is not require